git-replace, w: add pages (#4496)
tldr-pages/tldr: pages/common/git-replace.md

# git replace

> > Create, list, delete refs to replace objects.

Reviewer: Perhaps this will read better?
```suggestion
> Create, list, and delete refs to replace objects.
```

Author: haha silly mistakes

> > More information: <https://linux.die.net/man/1/git-replace>.

Reviewer 2: The official doc is better:
```suggestion
> More information: <https://git-scm.com/docs/git-replace>.
```

> - Replace any commit with a different one, rest commits unchanged:

Reviewer: Perhaps this will read better?
```suggestion
- Replace any commit with a different one, leaving other commits unchanged:
```
